pkg/controller/controller.go

```diff
@@ -50,13 +50,20 @@ func Run() error {
 	return nil
 }
 
-// Register to get callbacks for updates to objects
-// All handlers need to be registered before calling Run()
-func Register(
+// Register registers a controller with empty label selector.
+func Register(gkv *schema.GroupVersionKind, namespace string, resyncPeriod time.Duration, handler sdk.Handler) error {
```

Comment:
Line it too long with the params, put them on different lines like below

Reply:
Updated.